src/prompts/index.tsx

```diff
@@ -413,38 +297,16 @@ export class PromptManager {
         }
     }
 
-    private async loadPrompts(): Promise<void> {
-        this.platformConfig = await loadPlatformConfig(this.context);
-
-        if (!this.platformConfig.publicKey) {
+    private loadPrompts(): void {
```

Comment:
I think we can remove `loadPrompts` entirely now, we don't support anything other than 'auto' as a strategy for Optimove.

Reply:
Removed